.github/workflows/test.yml

```
@@ -0,0 +1,28 @@
+name: Tests
+
+on:
+  push:
+    branches: [ master, main ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)